example/app/app.js

```diff
@@ -33,6 +33,7 @@ app.get('/', function(req, resp) {
     ps[0] = gotsent.mostPopular();
     ps[1] = gotsent.mostHated();
     ps[2] = gotsent.mostDiscussed();
+    gotsent.startStream();
```

Comment:
You certainly don't want to start a new stream every time someone visits the main page.